app/models/batch.rb

```diff
@@ -634,12 +627,9 @@ def generate_target_assets_for_requests # rubocop:todo Metrics/AbcSize
 
       request.update!(target_asset: target_asset)
 
-      # All links between the two assets as new, so we can bulk create them!
-      asset_links << [request.asset.labware.id, target_asset.labware.id]
+      target_asset.parents << request.asset.labware
     end
 
-    AssetLink::BuilderJob.create(asset_links)
```

Comment:
We used to build asset links asynchronously, but as we now need them for looking up PhiX I've made them synchronous.